test-configs.yaml

```diff
@@ -780,7 +780,7 @@ device_types:
       guestfs_interface: 'virtio'
       machine: 'virt,gic-version=3'
     filters:
-      - whitelist: {defconfig: ['multi_v7_defconfig', 'vexpress_defconfig']}
+      - whitelist: {defconfig: ['multi_v7_defconfig']}
```

Comment:
Are we sure we still want to boot it on `qemu_arm-virt-gicv2` even though we don't currently know whether it's actually supposed to be booting?

Reply:
I have just checked, and vexpress_defconfig dont have ARCH_VIRT. Same for the final .config produced from it. So I will remove also gicv2